src/main/java/gtPlusPlus/xmod/gregtech/common/covers/CoverOverflowValve.java

```diff
@@ -66,7 +66,7 @@ protected OverflowValveData doCoverThingsImpl(ForgeDirection side, byte aInputRe
         OverflowValveData data, ICoverable aTileEntity, long aTimer) {
         if (data == null) return new OverflowValveData(maxOverflowPoint, maxOverflowPoint / 10, true, true);
 
-        if (data.voidingRate == 0) return data;
+        if (data.overflowPoint == 0) return data;
```

Comment:
If you have decided to prevent the cover from working when the overflow value is set to zero then you also should disable it when voiding rate equals 0. Since with zero voiding rate this cover will void nothing anyway.
```suggestion
        if (data.overflowPoint == 0 || data.voidingRate == 0) return data;
```

Reply:
@OneEyeMaker I hope it's finally done